nl_server/routes.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from flask import Blueprint
+
+bp = Blueprint('main', __name__, url_prefix='/')
+
+
+@bp.route('/')
```

Comment:
@jehangiramjad You can update this function, rename this file or add new files for the NL endpoints

Reply:
got it, thanks!

Reply:
my understanding is that this is strictly an API server to host bigger embeddings? in that case, we should perhaps set up with a /api endpoint.

Reply:
Yep. @jehangiramjad will need to update those and we can make sure they are /api/...